common/components/stores/ProjectSearchStore.js

```diff
@@ -171,10 +179,19 @@ class ProjectSearchStore extends ReduceStore<State> {
   }
 
   _addTagToState(state: State, tag: string): State {
+    //tag does not have to be a string, this works for an array of strings as in ADD_MANY_TAGS
     const newTags:$ReadOnlyArray<string> = state.tags.concat(tag);
     return state.set('tags', newTags);
   }
 
+  _removeTagsFromState(state: State, tags: array): State {
```

Comment:
I was happy to find the existing ADD_TAG function worked just fine to add multiple tags at once instead of just one, but I haven't tested the REMOVE_TAG for the same. Worth a look just so we can keep our code clean.